es5-shim.js

```diff
@@ -1654,7 +1764,7 @@ defineProperties(StringPrototype, {
     }
 }, hasTrimWhitespaceBug);
 
-var hasLastIndexBug = StringPrototype.lastIndexOf && 'abcあい'.lastIndexOf('あい', 2) !== -1;
+var hasLastIndexBug = StringPrototype.lastIndexOf && 'abc\u3042\u3044'.lastIndexOf('\u3042\u3044', 2) !== -1;
```

Comment:
Ok, altered some settings in my editor and changed them back, now I'm seeing a warning `this character may get silently deleted by one or more browsers`. Just thought I'd mention it before I go and disable this warning.